docs/content/en/command-line-interface/cli.md

```diff
@@ -0,0 +1,137 @@
+---
+date: 2022-3-25
+title: Command Line Interface
+draft: false
+---
+
+<!--Copyright (c) Coherent Labs AD. All rights reserved. Licensed under the MIT License. See License.txt in the project root for license information. -->
+
+The coherent-guic-cli is a tool that helps you create and develop components from the GameUIComponents suite. It can also setup a development environment with live reload that will allow you to easily create a UI using the components.
+
+# Getting Started
+
+To install the cli open a console and type:
+
+```
+npm i -g coherent-guic-cli
+```
+
+This will install the cli as a global module and will make it available from everywhere.
+
+## Available Commands
+
+|Command   |Description   |Arguments   |Usage   |
+|---|---|---|---|
+|create-project               |Create a new project from template                 | `<template>` `<directory>`|`coherent-guic-cli create-project Webpack ./my-folder`|
+|create | [name], [directory] |           Create a new component | `coherent-guic-cli create my-component ./my-folder`|
+|build |[--watch]        | start the server|`coherent-guic-cli build`|
+|start:demo| N/A|              Start a development server and host the demo|`coherent-guic-cli start:demo`|
+|build:demo| [--dev] |      Create a production bundle of the demo.
+
+
+## Usage
+
+### Setting up an Environment
+
+If you want to create a UI using the components - use the `create-project` command. It allows you to setup a project that uses [Webpack](https://webpack.js.org/) or [Vite](https://vitejs.dev/). Open a console and type:
+```
+coherent-guic-cli create-project Webpack ./my-folder
+```
+
+This will automatically install all dependencies. After the project is ready - navigate to `my-folder` and type
+```
+npm run dev
+```
+
+this will start a development server with live reload enabled.
+
+`<template>` can be either 'webpack' or 'vite'.
+
+Example:
+```
+coherent-guic-cli create-project vite ./my-folder
+```
+
+This will create a new project inside `./my-folder` and will install the dependencies. After that you can navigate to the folder and start a development server:
+```
+cd ./my-folder
+npm run dev
+```
+
+### Creating a Component
+After the installation is complete open a console and run:

```

Comment:
Installation of what exactly - the cli? Run, where? :D my-folder? :D We are a bit confused here because you are giving an example above of creating a project and maybe it should continue here with creating a new component to the new project but the context of this topic is different.

Reply:
Yep, it used to make some kind of sense before the changes, but now that I added the docs about create-project it is confusing. Added clarification about what this section is about - check it and let me know if it's still confusing.